llvm/test/TableGen/dag-subst.td

```diff
@@ -0,0 +1,18 @@
+// RUN: llvm-tblgen %s | FileCheck %s
+
+// Operator name "frag" in one_frag:$frg was not retained during !subst.
```

Comment:
we have operator `one_frag` named `frg`, but there is nothing that is just `frag`. Does the comment need fixing?